docs/release_notes/pending_release.rst

```diff
@@ -1,10 +1,17 @@
-SMQTK Pending Release Notes
+Pending Release Notes
 ===========================
 
 
 Updates / New Features
 ----------------------
+CI
 
+* Added workflow for CI on GitHub.
+
+Misc.
+
+* Updated various type annotations for type-checking compliance.
+* Updated .gitignore away from the old monorepo version.
```

Comment:
RST and Sphinx are a little picky about bullets: it will want new-lines in between. I think it's "fine" in this isolated case, but as soon as sub-bullets get added, the transpiler starts to complain more.
```suggestion
* Updated various type annotations for type-checking compliance.

* Updated .gitignore away from the old monorepo version.
```